flytekit/clis/sdk_in_container/register.py

```diff
@@ -194,5 +202,9 @@ def register(
         version = remote._version_from_hash(md5_bytes, serialization_settings, service_account, raw_data_prefix)  # noqa
         cli_logger.info(f"Computed version is {version}")
 
+    cli_logger.warn(
```

Comment:
can we use info or lower?  if this is something we want to print then click.echo?

Reply:
this should be printed. I'm going to change it to `click.echo`.